resources/zib/zib-HeartRate.xml

```diff
@@ -36,7 +36,7 @@

```

Comment:
'Several concept(s)' should be changed to 'Several concepts' (i.e. no parentheses).